pyproject.toml

```diff
@@ -17,6 +17,7 @@ max-line-length = "88"
 max-attributes = 15
 max-args= 6
 max-locals = 16
+good-names = "f"
```

Comment:
This is for it doesn't complain about you calling `f` the file handler? What about just calling it `fh`?

Reply:
It wants a name with at least 3 characters. I could use `file` (which seems ok to use in Python 3) but since it was used by Python in the past and my syntax highlighter still colors it differently, just using the canonical variable `f` looked more appropriate.

Reply:
Or I can use `yaml_file` as @sergiusens suggested below, will do it.